### ISSUE TITLE
JSON response instead of XML

### ISSUE THREAD

**Issue by marcoscheel** [SciDevs/delicious-api#37]
Is there a way to get JSON repsonses instead of XML? Any planes to support this on the here documentet set of API? There is already JSON if we access the oAuth stuff, but all the traditional API is only returning XML.

Thank you.

**Comment by peoplemerge** [comment]
Yes, this is absolutely in the plans!  Most of the backend work is already done, but the documentation will come after.

Hint: do an inspect element on delicious.com and watch the AJAX (err AJAJson?) calls.  You might hit some CORS issues on calls that protect customer data if you want to do it all in frontend, so the work remanining is to allow oAuth paths for those.

**Comment by marcoscheel** [comment]
I've tried the reverse engineering, but the main problem is the auth part. You don't rely on pure oAuth for the API on delicious.com :( Some kind of internal session tracking in your endpoint is to much of a hassel to really care about. I hope a developer version will arrive in the near future. I want to do a rewrite of my Windows 8 app once all the details for Windows 10 are anounced in the next few weeks.

**Comment by peoplemerge** [comment]
True, oAuth is only necessary for calls we make to auth via FB/ TW / G+.  However you can develop an oAuth app and use our oAuth endpoint.

When making calls to our oAuth implementation too, you should not need a session, just an oAuth token.  The other calls to avosapi.delicious.com (that you can see via inspect element) should be viable.

If they are not working or are incomplete, please elaborate with specifics, flows, endpoints, and we'll fix it.
